pkg/deploy/dev-workspace/dev_workspace.go

```diff
@@ -62,19 +61,23 @@ func NewDevWorkspaceReconciler() *DevWorkspaceReconciler {
 }
 
 func (d *DevWorkspaceReconciler) Reconcile(ctx *chetypes.DeployContext) (reconcile.Result, bool, error) {
-	if isDevWorkspaceOperatorCSVExists(ctx) {
```

Comment:
I was thinking that as a workaround we would improve this check e.g. 
`ifCSVExists || devworkspaceDeploymentExists { Do nothing }`

Reply:
`ifCSVExists` is not robust, so I've got rid of it.
And I added `isDevWorkspaceOperatorHasOwner` func to check if DWO deployment exists and managed by OLM.